src/validations/rules/ssp.sch

```diff
@@ -796,11 +795,15 @@
                 id="leveraged-PE-controls"
                 role="warning"
                 test="
-                    if ($leveraged/@uuid eq $component-ref and $statementID eq 'pe')
+                    if ($statementID eq 'pe')
                     then
-                        false()
+                        if ($leveraged/@uuid eq $component-ref)
+                        then
+                            true()
+                        else
+                            false()
                     else
-                        true()">This PE Control has a leveraged authorization - <xsl:value-of
+                        true()">This PE Control has a matching leveraged authorization - <xsl:value-of
                     select="../@statement-id" />.</sch:assert>
```

Comment:
This assertion should be deleted because the context is not `context="/oscal:system-security-plan/oscal:control-implementation/`. (That is already handled by `leveraged-PE-controls-implemented-requirement`.)